### PR TITLE
update keycloak image repo

### PR DESCRIPTION
**Pull request by Adam-D-Lewis**
## Reference Issues or PRs
Fixes https://github.com/nebari-dev/nebari/issues/2311.  We should test how this will affect users upgrading Nebari.  Ideally, the images are identical so there is no ill effects, but that may not be the case.

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request ~locally~ on azure?
- [ ] Did you add new tests?

### CODE REVIEW

File: src/_nebari/stages/kubernetes_services/__init__.py [hunk 1]
```diff
@@ -51,9 +51,15 @@ class Storage(schema.Base):
...
-    favicon: str = "https://raw.githubusercontent.com/nebari-dev/nebari-design/main/symbol/favicon.ico"
+    welcome: str = (
+        """Welcome! Learn about Nebari's features and configurations in <a href="https://www.nebari.dev/docs">the documentation</a>. If you have any questions or feedback, reach the team on <a href="https://www.nebari.dev/docs/community#getting-support">Nebari's support forums</a>."""
+    )
+    logo: str = (
+        "https://raw.githubusercontent.com/nebari-dev/nebari-design/main/logo-mark/horizontal/Nebari-Logo-Horizontal-Lockup-White-text.svg"
+    )
+    favicon: str = (
+        "https://raw.githubusercontent.com/nebari-dev/nebari-design/main/symbol/favicon.ico"
+    )
```
Comment (Adam-D-Lewis) [inline src/_nebari/stages/kubernetes_services/__init__.py:62]: unrelated formatting changes

**Review by dcmcand — APPROVED** [review]
Thanks for this @Adam-D-Lewis :rocket:
